src/tm-threads.c

```diff
@@ -1873,18 +1878,37 @@ TmEcode TmThreadWaitOnThreadInit(void)
                 return TM_ECODE_FAILED;
             }
 
-            if (i == TVT_MGMT)
-                mgt_num++;
-            else if (i == TVT_PPT)
-                ppt_num++;
+            if (active_runmode == RUNMODE_AFP_DEV) {
```

Comment:
This limits the support for this type of output to just afpacket, while other capture methods (should) follow the same pattern.

Reply:
![Screenshot from 2020-07-17 11-02-23](https://user-images.githubusercontent.com/32410611/87768833-08795e80-c81d-11ea-84cc-0c6e2bb5e8f1.png)
Adding NFQ example.